.github/workflows/ci.yaml

```diff
@@ -31,6 +31,18 @@ jobs:
       - name: PHP lint
         run: "find *.php Classes Configuration Tests -name '*.php' -print0 | xargs -0 -n 1 -P 4 php -l"
 
+  documentation-rendering:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Documentation without warnings
+        run: |
+          mkdir -p Documentation-GENERATED-temp \
+          && docker run --rm --pull always -v $(pwd):/project \
+             ghcr.io/typo3-documentation/render-guides:latest --config=Documentation --no-progress --fail-on-log
+
```

Comment:
Does it make sense to do this for every PR? OK, you can see if there are errors during rendering, but you can't download and check them here, can you?
Have you tested whether the task fails if there is a rendering problem?

Reply:
It would of course be better only to run it if there were changes in the `/Documentation` directory. But I did not have this idea before.
You can see the errors in the log, the will be printed to the console so it's not useless.

I have not tested it but I am quite sure it works because it's 1:1 copied from the [official documentation](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/Howto/RenderingDocs/Index.html#introduce-automatic-testing-for-extension-documentation).